website/static/js/nodeSelectTreebeard.js

```diff
@@ -78,7 +133,7 @@ function NodeSelectTreebeard(divID, data, nodesState) {
         }
     });
     var grid = new Treebeard(tbOptions);
-    projectSettingsTreebeardBase.expandOnLoad.call(grid.tbController);
+    expandOnLoad.call(grid.tbController);
```

Comment:
This call can be made with treebeard's existing hook : onload, unless that's already defined. The manual changes we made to 0.66 were like this. https://github.com/CenterForOpenScience/osf.io/blob/develop/website/static/js/nodesPrivacySettingsTreebeard.js#L71